initial-promises/node-server/common/1.0/cf-served.cf

```diff
@@ -58,6 +58,11 @@ bundle server access_rules
         maproot => { host2ip("${server_info.cfserved}"), string_downcase(escape("${server_info.cfserved}"))  },
         admit   => { host2ip("${server_info.cfserved}"), string_downcase(escape("${server_info.cfserved}"))  };
 
+    any::
+      # Allow server to remotely run the agent
+      "/var/rudder/cfengine-community/bin/cf-agent"
+        admit   => { host2ip("${server_info.cfserved}"), string_downcase(escape("${server_info.cfserved}")) };
```

Comment:
Because cf-serverd asks its hostname to the client before accepting the connection.